cmake/std/atdm/README.md

```diff
@@ -675,6 +676,11 @@ ATDM Trilinos configuration and testing scripts should work with just about
 any version of git 2.0+.  Another approach would be for the user to
 (temporarily) edit their `~/.gitconfig` file to address the problems.
 
+Another problem occurs for developers who don't use bash but instead need to
+switch into a bash shell.  In this case, one must use `bash -l` so that the
+users `.bash_profile` file will get sourced.  Without this, one can't load the
```

Comment:
user's

Reply:
@jmgate, I fixed in an amended commit and force-pushed.  Therefore, I dismissed your "request changes" review.